.circleci/config.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           command: yarn check-coverage #Validate code is adequately covered by unit tests
       - persist_to_workspace:
           root: ~/repo
-          paths: .
+          paths: [.]
```

Comment:
did you confirm that this is the right syntax here? not something like `["."]` instead, for example

Reply:
the CI is not showing a type issue there anymore. so I guess it is... :)